src/main/java/hudson/remoting/Util.java

```diff
@@ -100,63 +91,6 @@ static String indent(String s) {
         return "    " + s.trim().replace("\n", "\n    ");
     }
 
-    /**
```

Comment:
Fret not that these public methods are being deleted, as this whole class is `@Restricted(NoExternalUse.class)`. Anyway, these were all duplicate code. Keeping the more widely used copy.